src/models/things.ts

```diff
@@ -45,15 +47,15 @@ const Things = {
    * added is when the addon discovers a thing, not when the model is
    * instantiated
    */
-  emitter: new EventEmitter(),
+  private emitter = new EventEmitter();
```

Comment:
Side note: I tried taking this out and just making the Things class extend EventEmitter and it failed miserably. Not sure why.